.github/workflows/mdk-cli-cordova.yml

```diff
@@ -0,0 +1,29 @@
+name: Docker Image cordova
+
+on:
+  push:
+    branches: 
+    - master   
+    paths:
+    - mdk-cli-cordova/**
+  pull_request:
+    branches: 
+    - master   
+    paths:
+    - mdk-cli-cordova/**
```

Comment:
@p0l0us this should be removed before merge this PR, because you will push Docker image on each change in the PR